src/views/preview/project-not-available.jsx

```diff
@@ -0,0 +1,24 @@
+const React = require('react');
+const FormattedMessage = require('react-intl').FormattedMessage;
+const FlexRow = require('../../components/flex-row/flex-row.jsx');
+
+require('./project-not-available.scss');
+
+const ProjectNotAvailable = () => (
```

Comment:
The language here is generic - the same text and image used on every scratchr2 404 page. It also looks like none of the code here is specific to the project page. Should this be separated into a generic 404 view, which can be required and used as needed?

Reply:
@towerofnix good point! Moved to a component